optuna_dashboard/app.py

```diff
@@ -185,6 +193,38 @@ def get_study_detail(study_id: int) -> BottleViewReturn:
         trials = get_trials(storage, study_id)
         return serializer.serialize_study_detail(summary, trials)
 
+    @app.get("/api/studies/<study_id:int>/param_importances")
+    @handle_json_api_exception
+    def get_param_importances(study_id: int) -> BottleViewReturn:
+        # TODO(chenghuzi): add support for selecting params and targets via query parameters.
+        response.content_type = "application/json"
+        study_name = storage.get_study_name_from_id(study_id)
+        study = Study(study_name=study_name, storage=storage)
+
+        trials = [trial for trial in study.trials if trial.state == TrialState.COMPLETE]
+        if len(trials) == 0:
+            return ""
+        evaluator = None
+        params = None
+        target = None
+        importances = optuna.importance.get_param_importances(
+            study, evaluator=evaluator, params=params, target=target
+        )
+        if target is None:
+            target_name = "Objective Value"
+
+        return {
+            "target_name": target_name,
+            "param_importances": [
+                {
+                    "name": i[0],
+                    "importance": i[1],
+                    "distribution": get_distribution_name(i[0], study),
+                }
+                for i in importances.items()
```

Comment:
[nits] A bit refactoring.

```suggestion
                    "name": name,
                    "importance": importance,
                    "distribution": get_distribution_name(name, study),
                }
                for name, importance in importances.items()
```